lib-compose/src/main/java/com/what3words/components/compose/maps/W3WMapDefaults.kt

```diff
@@ -121,14 +129,16 @@ object W3WMapDefaults {
 
     fun defaultGridLinesConfig(
         isGridEnabled: Boolean = true,
-        gridColor: Color = Color.LightGray,
+        gridColor: Color = Color(105, 127,  141, 70),
+        gridColorDarkMode: Color = Color(105, 127,  141, 70),
```

Comment:
Can you please try to define these colors in a separate place for better maintainance?